.github/workflows/benchmark_execution.yml

```diff
@@ -156,29 +156,24 @@ jobs:
           tar -xf ${BENCHMARK_TOOLS_ARCHIVE}
           echo "normal-benchmark-tools-dir=${BENCHMARK_TOOLS_DIR}/build/tools" >> "${GITHUB_OUTPUT}"
           echo "traced-benchmark-tools-dir=${BENCHMARK_TOOLS_DIR}/build-traced/tools" >> "${GITHUB_OUTPUT}"
+          echo "tracy-capture-tool=${BENCHMARK_TOOLS_DIR}/build-traced/tracy/iree-tracy-capture" >> "${GITHUB_OUTPUT}"
       - name: "Running benchmarks"
-        id: run
         env:
           BENCHMARK_CONFIG: ${{ steps.download-assets.outputs.benchmark-config }}
           IREE_DOCKER_WRAPPER: ./build_tools/github_actions/docker_run.sh
           IREE_NORMAL_BENCHMARK_TOOLS_DIR: ${{ steps.unpack-tools.outputs.normal-benchmark-tools-dir }}
           IREE_TRACED_BENCHMARK_TOOLS_DIR: ${{ steps.unpack-tools.outputs.traced-benchmark-tools-dir }}
+          IREE_TRACY_CAPTURE_TOOL: ${{ steps.unpack-tools.outputs.tracy-capture-tool }}
           IREE_DEVICE_NAME: ${{ env.DEVICE_NAME }}
           IREE_E2E_TEST_ARTIFACTS_DIR: ${{ env.E2E_TEST_ARTIFACTS_DIR }}
           IREE_RUN_CONFIG: run-config.json
           IREE_BENCHMARK_RESULTS: ${{ env.BENCHMARK_RESULTS_DIR }}/benchmark-results-${{ matrix.benchmark.device_name }}.json
+          IREE_BENCHMARK_TRACY: ${{ env.BENCHMARK_RESULTS_DIR }}/benchmark-tracy-${{ matrix.benchmark.device_name }}.tar.gz
```

Comment:
nit: this name could refer to a Tracy executable. How about `IREE_BENCHMARK_TRACY_CAPTURES` or `IREE_BENCHMARK_TRACES`?

Reply:
IREE_BENCHMARK_TRACES looks good.